terraform/shared/shared.tf.json.template.py

```diff
@@ -44,7 +44,7 @@ def conformance_pack(name: str) -> str:
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Sum',
                     filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}',
-                    threshold=12,
+                    threshold=6,
```

Comment:
@hannes-ucsc: "Let's make this 8 to allow for some discrepancies regarding when a day begins / ends."